### PR TITLE
#1627 - Old Nodes in Scale

### PR DESCRIPTION
**Pull request by mikenholt**
##### Checklist

- [x] migrations are included
- [x] tests are included

### Affected app(s)

 - Node
- Scheduler
### Description of change

Modified the mesos offers logic to update a new Node field (last_offer_received) every time an offer for a node was received. An evaluation is done every 10 seconds where a node that is not running jobs and has not received offers in 5 minutes will be marked as `is_active=False` and purged from the node list that lives in memory for the scheduler.

Removed one test as I don't see why a node would be marked as `is_active=False` and we would want that to persist after a node restarts.  If there is a reason why we need that let me know.

I didn't add `last_offer_received` to the serializer, but I can if desired. I see as an internal-to-Scale field only.

### CODE REVIEW

**Comment by mikenholt** [comment]
I'll fix the test when I get a chance.

File: scale/job/models.py [hunk 1]
```diff
@@ -1271,11 +1271,11 @@ def get_job_exes(self, job_id, started=None, ended=None, statuses=None, node_ids
             if 'RUNNING' in statuses:
                 # This is a special case where we have to use exclusion so that running executions (no job_exe_end) are
                 # included
-                exclude_statues = []
+                exclude_statuses = []
```
Comment (JohnPTobe) [inline scale/job/models.py]: take those statues down!

File: scale/scale/local_settings2.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# Zookeeper URL for scheduler leader election. If this is None, only a single not is used and election isn't performed.
+SCHEDULER_ZK = None
+
+# The full name for the Scale Docker image (without version tag)
+SCALE_DOCKER_IMAGE = 'geoint/scale'
+BROKER_URL = 'amqp://guest:guest@localhost:55672//'
+
+POSTGIS_TEMPLATE = 'template_postgis'
+
+DATABASES = {'default': dj_database_url.config(default='postgis://scale:scale@localhost:55432/scale')}
```
Comment (JohnPTobe) [inline scale/scale/local_settings2.py]: accidentally added while testing?

Comment (mikenholt) [reply scale/scale/local_settings2.py]: Yeah this doesn't belong.....

**Comment by cshamis** [comment]
I like it!

Random emoji!

🎳

**Pull request merged**